packages/searchbase/src/CompositeAggregationResults.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import { parseCompAggToHits } from './utils';
+
+class CompositeAggregationResults {
+  // An array of composite aggregations obtained from the applied aggs in options.
+  data: Array<Object>;
+
+  // useful when loading data of greater size
+  afterKey: Object;
+
+  // Raw aggregations returned by ES query
+  raw: Object;
+
+  constructor(data?: Array<Object>) {
+    this.data = data || [];
+  }
+
+  // An array of original hits obtained from the applied query.
+  get rawData() {
+    return this.raw.buckets || [];
```

Comment:
What if `raw` value is undefined (which will be a case initially), it's better to add a check.